qemu/tests/balloon_hotplug.py

```diff
@@ -14,11 +16,12 @@ def run(test, params, env):
 
     1) Boot up guest w/o balloon device.
     2) Hoplug balloon device and check hotplug successfully or not.
-    3) Do memory balloon.
-    4) Reboot/shutdown/migrate guest after hotplug balloon device(option)
-    5) Do memory balloon after guest reboot(option)
-    6) Unplug balloon device and check unplug successfully or not.
-    7) Reboot/shutdown/migrate guest after unplug balloon device(option)
+    3) Install balloon service and check its status in windows guests(option).
```

Comment:
Do we need to verify the driver or something? I'm not sure. Other parts look good to me.

Reply:
I add driver check step in this step as well.
Thanks.